lib/http-bridge/src/Cardano/Wallet/HttpBridge/Network.hs

```diff
@@ -224,7 +230,7 @@ rbNextBlocks bridge start = maybeTip (getNetworkTip bridge) >>= \case
         pure (epochBlocks ++ additionalBlocks)
     Nothing -> pure []
   where
-    nextStableEpoch ix = do
+    nextStableEpoch (EpochNo ix) = do
```

Comment:
In http-bridge there's a concept as `EpochIndex`, which I think is now synonymous with `EpochNo`, but which I'm ignoring since we might want to remove http-bridge.